### DISCUSSION TITLE
Use <dialog> for dialog component

### DISCUSSION THREAD

**Discussion by GreatState** [chakra-ui/zag#68] [Ideas]
Have you thought about using ```dialog``` for the Dialog component?
Dialog is quite well supported also polyfills available: https://github.com/GoogleChrome/dialog-polyfill

**Comment by segunadebayo** [comment]
Hi @GreatState. You can totally use the `dialog` component if you want. From the looks of it, it doesn't seem as customizable as the dialog from Zag.

If you want more control over the dialog you can use the `zag-js/dialog`

**Comment by segunadebayo** [comment]
Closing — the dialog architecture has been finalized. We use a custom implementation for broader control over focus management and accessibility.
